substrate/client/transaction-pool/src/fork_aware_txpool/mod.rs

```diff
@@ -209,14 +209,15 @@
 //! ### Light maintain
 //! The [maintain](#maintain) procedure can sometimes be quite heavy, and it may not be accomplished
 //! within the time window expected by the block builder. On top of that block builder may want to
-//! build few blocks in the raw, not giving the pool enough time to accomplish possible ongoing
+//! build few blocks in the row, not giving the pool enough time to accomplish possible ongoing
 //! maintain process.
 //!
 //! To address this, there is a [light version][`ready_at_light`] of the maintain procedure. It
 //! [finds the best view][find_best_view], clones it and prunes all the transactions that were
```

Comment:
Last nit, I realized that this description is a bit far from reality - my mistake. You could add here a link to newly added function from `ViewStore`.  IIRC `find_best_view` was never used in `ready_at_light`.

Reply:
Done here: [f0bc263](https://github.com/paritytech/polkadot-sdk/pull/8533/commits/f0bc263b91fdfdd4983f8c0f69817ddc7e9b0b16).